contracts/LobstersNft.sol

```diff
@@ -127,6 +127,7 @@ contract LobstersNft is ILobstersNft, LobstersNames, ERC721, VRFConsumerBase {
 
   function fulfillRandomness(bytes32 _requestId, uint256 _randomNumber) internal override {
     require(seed == 0, "SEED_ALREADY_GENERATED");
+    finalBaseURI = true;
```

Comment:
doesn't this invalidate `setbaseURI`' behavior? do we assume `setBaseURI` is always called before `fulfillRandomness`?

Reply:
It should be finalized before we reveal the seed otherwise the owner can change token metadata & images any time, so there would be no reason in randomization at all.